package-lock.json

```diff
@@ -4327,7 +4327,7 @@
         },
         "fast-deep-equal": {
           "version": "1.1.0",
-          "resolved": "http://registry.npmjs.org/fast-deep-equal/-/fast-deep-equal-1.1.0.tgz",
+          "resolved": "https://registry.npmjs.org/fast-deep-equal/-/fast-deep-equal-1.1.0.tgz",
```

Comment:
Can we remove these `http` to `https` changes?

Reply:
It's auto-generated but I can totally revert `package-lock.json` if you prefer.

Reply:
> It's auto-generated 

Ya it is an npm issue. It is a nit actually so if you are still on the same branch just revert the file otherwise never mind.

Reply:
Sure, no worries, file reverted.